ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -44,6 +44,10 @@ base_images:
     namespace: ocp
     tag: upi-installer
 releases:
+  firstz:
```

Comment:
call this `oldest-supported`

Reply:
`firstz` -> `oldest-supported-z-rollback` in 9f0e0dec87 -> 7e35f2921d.